src/zimscraperlib/zim/creator.py

```diff
@@ -170,27 +186,28 @@ def config_metadata(
                 "Flavour": Flavour,
                 "Source": Source,
                 "Scraper": Scraper,
+                "Illustration_48x48@1": Illustration_48x48_at_1,
             }
         )
-        self.metadata.update(extras)
-        language = self.metadata.get("Language", "").split(",")
+        self._metadata.update(extras)
```

Comment:
We should call `_validate_metadata` here

Reply:
So we should validate metadata before the `start()`?

Reply:
No, before the call to `super().__enter__()` which trigger the starting process of the ZIM (creates temp files, etc).

Reply:
Should we do the check in the `config_metadata` or in the `start()` but before the `super()__enter__()`?

Reply:
In the `start()` before the `__enter__()` yes